ffi/capi/tests/missing_apis.txt

```diff
@@ -59,11 +59,14 @@ icu::plurals::PluralElements::with_zero_value#FnInStruct
 icu::plurals::PluralElements::zero#FnInStruct
 icu::plurals::PluralOperands::is_exactly_one#FnInStruct
 icu::plurals::PluralOperands::is_exactly_zero#FnInStruct
-icu::properties::PropertyNames#Struct
-icu::properties::PropertyNames::new_long#FnInStruct
-icu::properties::PropertyNames::new_short#FnInStruct
-icu::properties::PropertyNamesBorrowed#Struct
-icu::properties::PropertyNamesBorrowed::get#FnInStruct
+icu::properties::PropertyNamesLong#Struct
```

Comment:
nit: potentially in a followup since you're doing a bunch of different changes here, but can you add all the annotations to keep this clean?

Reply:
These are actually missing APIs.